src/vm/eetwain.cpp

```diff
@@ -5356,6 +5371,18 @@ OBJECTREF EECodeManager::GetInstance( PREGDISPLAY    pContext,
     _ASSERTE(*castto(table, unsigned short *) == 0xBABE);
 #endif
 
+#else // WIN64EXCEPTIONS
+
+    // Untracked table must have at least one entry - this pointer
+    _ASSERTE(info.untrackedCnt > 0);
```

Comment:
@janvorli I checked all the call sites to `GetInstance()` - from `StackFrameIterator::NextRaw(void)` and `CrawlFrame::GetThisPointer()`. These call sites check if the function is static so it looks OK to assume that we always have "this" in the function. And I think the changes in lclvars.cpp ensures "this" pointer. "this" seems to be always lclNum=0 so it goes to the first entry.